sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -2059,3 +2059,82 @@ test_suite(
         for key in SIGVERIFY_KEY_TYPE_KEYS
     ],
 )
+
+otp_json(
+    name = "otp_json_sigverify_key_validity",
+    partitions = [
+        otp_partition(
+            name = "CREATOR_SW_CFG",
+            items = {
+                "CREATOR_SW_CFG_KEY_IS_VALID": "0x4b4b4b4b4b4b4b4b",
```

Comment:
nit: the testplan looks like it asks for setting this entry to a value "other than `kHardenedBoolTrue`", not necessarily `kHardenedBoolFalse`. Should we add some additional tests for an invalid value here?

Reply:
> This reminds me. For manufacturing purposes, do we have to enable the keys before we enable ROM execution?

We don't have to as long as we are in a `TEST_UNLOCKED*` state. `PROD`, `PROD_END`, `DEV`, and `RMA` filters keys based on this OTP item. We can tune the behavior based on the comments on the manufacturing docs.

> We should capture this in the manufacturing docs.

Yes, this will be in that doc (as soon as I get the chance to work on it again). 

> nit: the testplan looks like it asks for setting this entry to a value "other than kHardenedBoolTrue", not necessarily kHardenedBoolFalse. Should we add some additional tests for an invalid value here?

Good point, it would be nice to have but I think this is fine too for now.

Reply:
Actually, I just checked the code and we throw the same error so using different values will not complicate the test. I updated the entries of the first three keys to `0xff004b`.